src/features/auth/components/Login.tsx

```diff
@@ -45,12 +54,92 @@ export const Login: FC = () => {
     };
   };
 
+  const form = useForm<z.infer<typeof loginUser>>({
+    resolver: zodResolver(loginUser),
+    defaultValues: {
+      email: "",
+      password: "",
+    },
+  });
+
+  const moveToCreateAccount = () => {
+    navigate("/signup");
+  };
+
   // TODO: build the ui
   // register email and password input
   return (
-    <form onSubmit={handleSubmit(onEmailAndPassword)}>
-      {/* TODO: form inputs */}
-      <button onClick={onProvider(GoogleAuthProvider)}>google</button>
-    </form>
+    <div className="flex h-screen w-full items-center justify-center">
+      <div className="w-full max-w-sm rounded-lg border border-gray-200 bg-white p-4 shadow dark:border-gray-700 dark:bg-gray-800 sm:p-6 md:p-8">
+        <Form {...form}>
+          <form
+            onSubmit={handleSubmit(onEmailAndPassword)}
+            className="space-y-6 justify-self-center"
+          >
+            <h5 className="text-xl font-medium text-gray-900 dark:text-white">
+              Sign in to{" "}
+              <label
+                htmlFor=""
+                className="font-dancing-script text-3xl font-bold"
+              >
+                Blogify
+              </label>
+            </h5>
+            <FormField
+              control={form.control}
+              name="email"
+              render={({field}) => (
+                <FormItem className="mb-5">
+                  <FormLabel className="mb-2 block text-sm font-medium text-gray-900 dark:text-white">
+                    email
+                  </FormLabel>
+                  <FormControl>
+                    <Input
+                      placeholder="email@gmail.com"
+                      {...field}
+                      {..._register("email")}
+                      className="block w-full rounded-lg border border-gray-300 bg-gray-50 p-2.5 text-sm text-gray-900 focus:border-blue-500 focus:ring-blue-500 dark:border-gray-600 dark:bg-gray-700 dark:text-white dark:placeholder-gray-400 dark:focus:border-blue-500 dark:focus:ring-blue-500"
```

Comment:
Why are you using blue colors ? 
leave it as default for now, we'll decide about the palette colors later